### PR TITLE
added index.html

### PR DESCRIPTION
**Pull request by jonnylynchy**
Let me know if you want me to change anything!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Amazing Project!</title>
+</head>
```
Comment (jonnylynchy) [inline index.html]: I really like this <head> section man!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+<body>
+    <div>
+
+        <h1>Amazing Project</h1>
+        <p>This project is seriously amazing! Check it out y'all!</p>
```
Comment (justinkunz) [inline index.html]: GREAT H1 TAG!! PERFECT!!!!!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    </div>
+</body>
+
+</html>
```
Comment (justinkunz) [inline index.html]: ...seriously????? Fix this now!

Comment (jonnylynchy) [reply index.html]: I'm on it!

**Pull request merged**